internal/region/region.go

```diff
@@ -0,0 +1,62 @@
+package region
+
+// Region represents the sauce labs region.
+type Region uint
+
+const (
+	// None is an undefined sauce labs region.
+	None Region = iota
```

Comment:
Should we default to `us-west-1`?

Reply:
I thought about it, but we already default to us-west-1 in the config/arg parser.
However, I wanted to be able to represent a "no region selected" option via enums for other purposes, without abusing pointers for this.